bin/main.rb

```diff
@@ -1,90 +1,114 @@
 #!/usr/bin/env ruby
+# rubocop:disable Metrics/PerceivedComplexity,Metrics/CyclomaticComplexity
 
-# Welcome screen for the game
-p 'Hello World From Brazil and Colombia'
-puts '****************************************************************'
-puts '*                    TIC TAC TOE THE GAME                      *'
-puts '****************************************************************'
-print "\n\n"
-puts ' Welcome to the TIC TAC TOE game. This is a two player game so'
-puts ' make sure you have a friend to play with.'
-puts ' The rules are simple: The first player to get 3 tokens in line'
-puts ' wins. Bring your A game'
-print "\n\n"
-
-loop do
-  print 'Do you want to start the game?[y/n]: '
-  init = gets.chomp
-  if init == 'n'
-    puts 'Thank you for playing'
-    return
+class Game
+  attr_accessor :turn, :board
+  attr_reader :player1, :player2
+
+  def initialize(player1 = Player, player2 = Player, board = Board)
+    @player1 = player1
+    @player2 = player2
+    @board = board
+    @turn = 0
   end
-  break unless init != 'y' && init != 'n'
-end
 
-# Gets the information on the participants
-puts "\n\n Lets get to know our participants!!!!!!\n\n"
-player1 = ''
-player2 = ''
-init = ''
-loop do
-  print 'Enter the name of player #1: '
```

Comment:
- [ ] All the game logic should be in the lib folder. Also, put every class in a separate file (inside the lib folder)

![image](https://user-images.githubusercontent.com/61479648/90543598-522ad100-e15c-11ea-8610-6f0b17f7245d.png)